TestProject/ClientApp/src/components/Post.js

```diff
@@ -0,0 +1,16 @@
+﻿import React from 'react';
+
+function Post(props) {
```

Comment:
Use object destructing
```suggestion
function Post({ user, title, createDate, content  }) {
```